data-science-onramp/feature-engineering/citibike.ipynb

```diff
@@ -0,0 +1,385 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feature Engineering with Pandas and scikit-learn\n",
+    "\n",
+    "This notebook demonstrates how to use AI Platform notebooks to perform feature engineering on a dataset using `pandas`.\n",
+    "\n",
+    "You will load the data into a `Pandas DataFrame`, clean up the columns into a useable format, and then restructure the data into feature and target data columns. \n",
```

Comment:
double check there's only one space in between Pandas and Dataframe - it looks like two to me, but it also may just be that the different typefaces render spaces differently

Reply:
There's only one!